README.md

```diff
@@ -1 +1,34 @@
-# hello_microverse-project
\ No newline at end of file
+<div align="center">
+
+  <h3><b>Hello Microverse</b></h3>
+
+</div>
+
+> **Hello Microverse** is a project meant to practice linters on html and css, GitHub_flow, pull_request and more in order to use the learned in all Microverse's Projects.
+
+---
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- CSS
+
+---
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Hello Microverse](https://google.com)
+
+---
+
+## 👥 Author <a name="authors"></a>
+
+- GitHub: [@ITurres](https://github.com/ITurres)
+- Twitter: [@Arthur_ITurres](https://twitter.com/ArthurIturres)
+- LinkedIn: [Arthur Emanuel G. Iturres](https://www.linkedin.com/in/arturoemanuelguerraiturres/)
+
+---
+
+## 📝 License <a name="license"></a>
+
```

Comment:
- [ ] Your MIT link is broken. kindly try to fix it.we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.

Reply:
Got it! will fix. thank you tufoinnkuo10!